CHANGELOG.md

```diff
@@ -1,3 +1,270 @@
+  [#17508](https://github.com/npm/npm/pull/17508)
+  Add a new `requires` field to `package-lock.json` with information about the
+  _logical_ dependency tree. This includes references to the specific version
+  each package is intended to see, and can be used for many things, such as
+  [converting `package-lock.json` to other lockfile
+  formats](https://twitter.com/maybekatz/status/880578566907248640), various
+  optimizations, and verifying correctness of a package tree.
+  ([@iarna](https://github.com/iarna))
+* [`47e8fc8eb`](https://github.com/npm/npm/commit/47e8fc8eb9b5faccef9e03ab991cf37458c16249)
+  [#17508](https://github.com/npm/npm/pull/17508)
+  Make `npm ls` take package locks (and shrinkwraps) into account. This means
+  `npm ls` can now be used to see [which dependencies are
+  missing](https://twitter.com/maybekatz/status/880446509547794437), so long as
+  a package lock has been previously generated with it in.
+  ([@iarna](https://github.com/iarna))
+* [`f0075e7ca`](https://github.com/npm/npm/commit/f0075e7caa3e151424a254d7809ae4489ed8df90)
+  [#17508](https://github.com/npm/npm/pull/17508)
+  Take `package.json` changes into account when running installs -- if you
+  remove or add a dependency to `package.json` manually, npm will now pick that
+  up and update your tree and package lock accordingly.
+  ([@iarna](https://github.com/iarna))
+* [`83a5455aa`](https://github.com/npm/npm/commit/83a5455aac3c5cc2511ab504923b652b13bd66a0)
+  [#17205](https://github.com/npm/npm/pull/17205)
+  Add `npm udpate` as an alias for `npm update`, for symmetry with
+  `install`/`isntall`.
+  ([@gdassori](https://github.com/gdassori))
+* [`57225d394`](https://github.com/npm/npm/commit/57225d394b6174eb0be48393d8e18da0991f67b6)
+  [#17120](https://github.com/npm/npm/pull/17120)
+  npm will no longer warn about `preferGlobal`, and the option is now
+  deprecated.
+  ([@zkat](https://github.com/zkat))
+* [`82df7bb16`](https://github.com/npm/npm/commit/82df7bb16fc29c47a024db4a8c393e55f883744b)
+  [#17351](https://github.com/npm/npm/pull/17351)
+  As some of you may already know `npm build` doesn't do what a lot of people
+  expect: It's mainly an npm plumbing command, and is part of the more familiar
+  `npm rebuild` command. That said, a lot of users assume that this is the way
+  to run an npm `run-script` named `build`, which is an incredibly common script
+  name to use. To clarify things for users, and encourage them to use `npm run
+  build` instead, npm will now warn if `npm build` is run without any arguments.
+  ([@lennym](https://github.com/lennym))
+* [`d57d4f48c`](https://github.com/npm/npm/commit/d57d4f48c6cd00fdf1e694eb49e9358071d8e105)
+  [#17319](https://github.com/npm/npm/pull/17319)
+  Add new `--silent` option to `npm run-script` to suppress output from the
+  scripts themselves, since `--loglevel=silent` is only for `npmlog` itself.
```

Comment:
That's not what this commit does. This is purely a doc update to mention that you can suppress extra log output when running scripts with it. It does not suppress output from the scripts themselves.